### ISSUE TITLE
Does your code work ?

### ISSUE THREAD

**Issue by tirthankar95** [nurpeiis/LeakGAN-PyTorch#4]
During adversarial training the loss of manager and worker is not decreasing.

Adv-Manager loss: -0.08841 Adv-Worker loss: 2.23867
Adv-Manager loss: -0.09534 Adv-Worker loss: 2.27381
Adv-Manager loss: -0.08895 Adv-Worker loss: 2.30669
Adv-Manager loss: -0.07879 Adv-Worker loss: 2.29098
Adv-Manager loss: -0.10697 Adv-Worker loss: 2.26556
Adv-Manager loss: -0.08513 Adv-Worker loss: 2.25378
Adv-Manager loss: -0.09793 Adv-Worker loss: 2.22682
Adv-Manager loss: -0.08259 Adv-Worker loss: 2.33645
Adv-Manager loss: -0.10284 Adv-Worker loss: 2.35718
Adv-Manager loss: -0.09209 Adv-Worker loss: 2.24998
Adv-Manager loss: -0.10804 Adv-Worker loss: 2.25576
Adv-Manager loss: -0.08657 Adv-Worker loss: 2.27962
Adv-Manager loss: -0.08407 Adv-Worker loss: 2.31625
Adv-Manager loss: -0.09378 Adv-Worker loss: 2.26275
Adv-Manager loss: -0.10218 Adv-Worker loss: 2.28416
Adv-Manager loss: -0.09264 Adv-Worker loss: 2.23963
Adv-Manager loss: -0.09173 Adv-Worker loss: 2.20107
Adv-Manager loss: -0.09368 Adv-Worker loss: 2.25144
Adv-Manager loss: -0.09144 Adv-Worker loss: 2.20532
Adv-Manager loss: -0.10506 Adv-Worker loss: 2.26402
Adv-Manager loss: -0.08428 Adv-Worker loss: 2.22106
Adv-Manager loss: -0.09016 Adv-Worker loss: 2.18525
Adv-Manager loss: -0.08735 Adv-Worker loss: 2.31788
Adv-Manager loss: -0.09162 Adv-Worker loss: 2.29412
Adv-Manager loss: -0.09334 Adv-Worker loss: 2.26874
Adv-Manager loss: -0.08949 Adv-Worker loss: 2.32384
Adv-Manager loss: -0.08718 Adv-Worker loss: 2.30045
Adv-Manager loss: -0.10140 Adv-Worker loss: 2.29413
